creator/decorators.py

```diff
@@ -29,11 +29,22 @@ class task:
     A decorator to uniformly setup tasks that get enqueued for workers to
     execute.
 
-    If the wrapped task contains a release_id or task_id, save the log to the
-    appropriate model so that it may be retrieved relationally.
+    related_models may be specified to attach the generated job_log to other
+    models.
+    This takes one of two forms:
+        {Model1: "kwarg_name_for_pk", Model2: ...}
+            or
+        {Model1: ModelInstance, Model2: ...}
```

Comment:
What's the use case for this?

Reply:
So we can pass model instances in directly or by primary key:
```
release = Release()
my_job(release=release)
my_job(release=release.pk)
```
Both of these are valid.

Reply:
Oh oh I see. I kept imagining `task({Release: '182793628136'})` but that makes no sense